examples/go_examples.md

```diff
@@ -117,13 +117,13 @@ install:
   - go get -v github.com/codeclimate/test-reporter
   - cd $GOPATH/src/github.com/golang/dep/cmd/dep && git checkout tags/v0.4.1 && go install
   - cd $GOPATH/src/github.com/codeclimate/test-reporter && git checkout tags/v0.4.3 && go install
-  - cd - && dep ensure -v -vendor-only
+  - cd $GOPATH/src/github.com/user/project && dep ensure -v -vendor-only
 before_script:
   - test-reporter before-build
 script:
-  - go test -coverprofile c.out
+  - go test -coverprofile c.out -coverpkg ./...
 after_script:
-  - test-reporter after-build --input-type gocov --exit-code $TRAVIS_TEST_RESULT
+  - test-reporter after-build --cover-input-type gocov --exit-code $TRAVIS_TEST_RESULT
```

Comment:
Good catch! Although the flag is [`--coverage-input-type`](https://github.com/codeclimate/test-reporter/blob/master/cmd/after-build.go#L63).

Also, you are not need required to use the flag in this case. Since the test-reporter by default looks for [`c.out`](https://github.com/codeclimate/test-reporter/blob/master/formatters/gocov/gocov.go#L26). Using this flag is required when you store the output of `go test -coverprofile` on a different location.

Reply:
Yes, and it keeps confusing me as `after-build` and `format-coverage` use different switches to specify the input type.

I will remove this flag.

```
$ test-reporter after-build --help
[...]
  -t, --coverage-input-type string   type of input source to use [clover, cobertura, coverage.py, excoveralls, gcov, gocov, jacoco, lcov, simplecov]
[...]

$ test-reporter format-coverage --help
[...]
  -t, --input-type string   type of input source to use [clover, cobertura, coverage.py, excoveralls, gcov, gocov, jacoco, lcov, simplecov]
[...]
```

I can submit a PR for this too, but that's gonnna break a lot of builds for a lot of people.

Reply:
@jdel the flag names are intentional though. `after-build` is wrapper for two low level commands and some names might collide if we just use them as is.